stripe-ui-core/api/stripe-ui-core.api

```diff
@@ -93,6 +175,10 @@ public final class com/stripe/android/uicore/elements/SectionFieldElementUIKt {
 public final class com/stripe/android/uicore/elements/SectionUIKt {
 }
 
+public final class com/stripe/android/uicore/elements/SimpleTextFieldConfig$Companion {
+	public final fun getNAME ()Lcom/stripe/android/uicore/elements/SimpleTextFieldConfig;
```

Comment:
Is this used? Can we remove it? I don't think it's actually considered public, since `SimpleTextFieldConfig` is restricted.

Reply:
removed!